scripts/deploy.ts

```diff
@@ -100,16 +103,19 @@ async function main() {
 	await discordMessage(`--- **pre-flight complete, starting real deploy** ---`)
 
 	// 2. deploy the cloudflare workers:
-	await discordStep('[4/6] deploying asset uploader to cloudflare', async () => {
+	await discordStep('[4/7] deploying asset uploader to cloudflare', async () => {
 		await deployAssetUploadWorker({ dryRun: false })
 	})
-	await discordStep('[5/6] deploying multiplayer worker to cloudflare', async () => {
+	await discordStep('[5/7] deploying multiplayer worker to cloudflare', async () => {
 		await deployTlsyncWorker({ dryRun: false })
 	})
+	await discordStep('[6/7] deploying health worker to cloudflare', async () => {
+		await deployHealthWorker({ dryRun: false })
```

Comment:
you should add a `dryRun: true` version up above in the 'cloudflare deploy dry run' zone

Reply:
fixed